views/page/course/module/add-module.html

```diff
@@ -20,103 +20,93 @@
-                            value: "file",
-                            text: "File"
-                            },
-                            {
-                            value: "face-to-face",
-                            text: "Face-to-face"
-                            },
-                            {
-                            value: "video",
-                            text: "Video"
-                            }
-                        ]
-                        }) }}
-                        {{ govukButton({
-                            text: "Add item",
-                            classes: "button-blue"
-                        }) }}
-
-                    </fieldset>
-                </form>
-
-                {{ govukButton({
-                    text: "Continue",
-                    type: "button"
-                }) }}
-
-            <a class="cancel-button govuk-body govuk-link" href="#">Cancel</a>          
-        </div>
-        <div class="govuk-grid-column-one-third">   
+                            {{ govukSelect({
+                                id: "modules",
+                                name: "module",
+                                classes: "govuk-!-width-one-half govuk-!-margin-bottom-6",
+                                label: {
+                                    text: "Add a new module" 
+                                },
+                                items: [
+                                    {
+                                    value: "",
+                                    text: "Type of learning",
+                                    selected: true
+                                    },
+                                    {
+                                    value: "link",
+                                    text: "Link"
+                                    },
+                                    {
+                                    value: "file",
+                                    text: "File"
+                                    },
+                                    {
+                                    value: "face-to-face",
+                                    text: "Face-to-face"
+                                    },
+                                    {
+                                    value: "video",
+                                    text: "Video"
+                                    }
+                                ]
+                            }) }}
+                            {{ govukButton({
+                                text: "Add item",
+                                classes: "button-blue"
+                            }) }}
+                        </fieldset>
+                    </form>
+                <div class="button-container">
+                  <a class="govuk-button" href="#">Continue</a>          
+                </div>
+                <a class="cancel-button govuk-body govuk-link" href="#">Cancel</a>          
```

Comment:
Can you sort out the value of this href? It should probs bring user back to course preview